### PR TITLE
Small qol improvements

### PR DESCRIPTION
**Pull request by LeonVreling**
- Displays nationalities instead of countries when the question field is nationality -> i.e. "Dutch" instead of "The Netherlands"
- Sorts events on the dashboard in chronological order instead of alphabetical, which is more logical when you already have made a shift in dates (past and future)
- Fixes something that caused constant warnings on the console during testing, just to make it easier to use the console 😄

### CODE REVIEW

File: src/components/layout/Sidebar.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
         {{ category.categoryName }}
       </p>
       <ul class="menu-list">
-        <li v-for="item in category.components" exact="true" v-bind:key="item.name">
```
Comment (serge1peshcoff) [inline src/components/layout/Sidebar.vue:8]: this one is included in many places actually, I will remove it later everywhere

Comment (LeonVreling) [reply src/components/layout/Sidebar.vue:8]: Ah yeah these were the two places where it was constantly complaining, but I did not check for others as they didn't bother me that much 😅

**Comment by LeonVreling** [comment]
Oh wait it is missing a commit

**Comment by LeonVreling** [comment]
As per #MEMB-839, the link to the CIA on the resource page was old and did not actually link to the right page

File: src/views/static/Resources.vue [hunk 2]
```diff
@@ -159,7 +159,7 @@ export default {
             color: '#A0C514'
           },
           {
-            url: 'https://www.zeus.aegee.org/portal/new-cia-29-1/',
+            url: 'https://www.zeus.aegee.org/portal/cia-30-0/',
```
[the remark this answers is not in the document]
Comment (LeonVreling) [reply src/views/static/Resources.vue]: I updated the link, you were right 😅

**Comment by serge1peshcoff** [comment]
Will merge and deploy tomorrow, @LeonVreling is it ready to merge?

**Comment by LeonVreling** [comment]
Yes I suppose it is ready

**Pull request merged**